### PR TITLE
python test clean up

### PR DESCRIPTION
**Pull request by kensipe**
1. removed python file references that no longer exist
2. changed comments which were references to older code which were refactored.

### CODE REVIEW

**Comment by cmaloney** [comment]
cc: @mellenburg

**Comment by mellenburg** [comment]
:+1:  LGTM thanks for picking up litter!

File: test_util/test_runner.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-"""Module for running integration_test.py inside of a remote cluster
+"""Module for running integration_tests inside of a remote cluster
```
Comment (branden) [inline test_util/test_runner.py]: Was that supposed to be a space instead of an underscore?

**Comment by mellenburg** [comment]
@orsenthil Yeah, that can also go. test_registry was totally removed when we started running the test_server via a docker volume mount

**Comment by orsenthil** [comment]
@kensipe - opportunity in this PR. Please `git rm -r test_util/docker` and squash commits for the change.

**Comment by cmaloney** [comment]
🚢 . I did the follow up commented here in #535

**Pull request merged**